src/olympia/abuse/tests/test_views.py

```diff
@@ -0,0 +1,170 @@
+import json
+
+from django.core import mail
+from django.core.urlresolvers import reverse
+
+from olympia import amo
+from olympia.abuse.models import AbuseReport
+from olympia.amo.tests import (
+    addon_factory, APITestClient, TestCase, user_factory)
+
+
+class AbuseReviewSetTestBase(object):
+    client_class = APITestClient
+
+    def setUp(self):
+        self.url = reverse('abusereport-list')
+
+    def check_reporter(self, report):
+        raise NotImplementedError
+
+    def check_report(self, report, text):
+        assert unicode(report) == text
+        assert report.ip_address
```

Comment:
Can you actually check the value? You should be able to pass a custom `REMOTE_ADDR` to your `client.post()` calls.